### ISSUE TITLE
Extra spaces in fediverse => Bluesky bio

### ISSUE THREAD

**Issue by Cellosia** [snarfed/bridgy-fed#1557]
This is a very minor, silly issue, but whenever a bio gets truncated it'll get ellipsized with `  [...]  `. Note the extra space after the closing bracket.

There's another extra space in the bridging disclaimer: `[bridged from URL on the fediverse by fed.brid.gy ]`

When you're limited to 256 characters every lost character matters.

On a related note I'm not sure if it's necessary to specify it was bridged from the fediverse (or other services) when already providing an url. It's good advertisement for the original service, but it eats other 16 characters.

**Comment by jfietkau** [comment]
> This one is sadly necessary. Without it, Bluesky's auto-linking logic includes the trailing `]` and links to [https://fed.brid.gy/]](https://fed.brid.gy/%5D) , which doesn't work.

You could presumably use a [zero-width space](https://en.wikipedia.org/wiki/Zero-width_space) (U+200B) instead of a standard space character here. It wouldn't affect the character count, but would look slightly nicer.

**Comment by snarfed** [comment]
@jfietkau good idea! Sadly I tried it just now and it doesn't work, bsky.app's auto-linker ignores the ZWS and included the characters after it in the link.

**Comment by snarfed** [comment]
Done! Switched to using the 🌉 emoji, ie:

> 🌉 bridged from [bsky.app/profile/snarfed.bsky.social](https://bsky.app/profile/snarfed.bsky.social) on Bluesky by [fed.brid.gy](https://fed.brid.gy/)

Suggested by someone on here, but I can't find who/where. Thank you anyway!
